src/express/api.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const axios = require(`axios`);
+
+const TIMEOUT = 1000;
+
+const port = process.env.API_PORT || 3000;
+const defaultUrl = `http://localhost:${port}/api/`;
+
+class API
+{
+    constructor(baseURL, timeout)
+    {
+        this._http = axios.create(
+            {
+                baseURL,
+                timeout
+            }
+        );
+    }
+
+    async _load(url, options)
+    {
+        const response = await this._http.request({ url, ...options });
+        return response.data;
+    }
+
+    async getArticles() {
```

Comment:
```suggestion
   getArticles() {
```
если не используешь await внутри, тебе это не нужно. Но писать можно да, для наглядности. Если хочешь, то можешь везде исправить